lib/auth.coffee

```diff
@@ -222,7 +231,7 @@ getAuth = (deps, opts) ->
 	# });
 	###
 	exports.getToken = (callback) ->
```

Comment:
I think this should probably become `getKey`, and the docs should change `token` to something like `raw API key or session token`.

Reply:
I'm changing it to ```getKey``` but needs to change it also in the UI: https://github.com/resin-io/resin-ui/issues/1582